internal/sliceutils/sliceutils.go

```diff
@@ -47,3 +50,25 @@ func intsContain(ns []int, v int) bool {
 func InsertStringAt(data []string, i int, v string) []string {
 	return append(data[:i], append([]string{v}, data[i:]...)...)
 }
+
+func Push[T any](data []T, v T) []T {
+	return append(data, v)
+}
```

Comment:
Sorry, but this one is just replacing a native language API with effectively an alias that's opinionated towards a different language API. There's no new logic here. Please drop.